src/authentication-method.ts

```diff
@@ -6,19 +6,20 @@ import type { AuthenticatorCredentials } from "./user";
 export async function authenticationMethodFromEnv(env: Env) {
   if (env.JWT_REGISTRY_TOKENS_PUBLIC_KEY) {
     return await newRegistryTokens(env.JWT_REGISTRY_TOKENS_PUBLIC_KEY);
```

Comment:
nit: can we make this be
`(env.USERNAME && env.PASSWORD) || (env.READONLY_USERNAME && env.READONLY_PASSWORD)`

IK the logic is correct but just trying to make it a bit more obvious that it's doing the right thing first hand.